sdk/storage/Azure.Storage.DataMovement/src/TransferProperties.cs

```diff
@@ -13,46 +14,55 @@ public class TransferProperties
         /// <summary>
         /// Contains the transfer ID which to rehydrate the StorageResource from.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Maybe since we are handing these back to customers when they call `GetTransfers` we should have some info visible. Maybe `TransferId`, `SourceUri`, and `DestinationUri`?